heareval/tasks/pipeline.py

```diff
@@ -441,12 +441,20 @@ def run(self):
             )
 
             if self.data_config["embedding_type"] == "scene":
-                # For scene labeling each scene should have one
-                assert len(audiolabel_df) == len(audiodf)
+                # For scene labeling each scene has a list of labels. For multiclass
+                # predictions, the list will contain exactly one label, for multilabel
+                # the list will can zero or more labels.
+                # TODO: not sure how we do multilabel when there are zero labels.
+                if self.data_config["prediction_type"] == "multiclass":
+                    # For multiclass there should be exactly one audio file per label
+                    assert len(audiolabel_df) == len(audiodf)
+
+                # Create a dictionary containing a list of labels keyed on the slug.
                 audiolabel_json = (
                     audiolabel_df[["slug_path", "label"]]
-                    .set_index("slug_path")
-                    .to_dict("index")
+                    .groupby("slug_path")["label"]
+                    .apply(list)
+                    .to_dict()
```

Comment:
Before this was:
```
{'slug_name': {'label': 'dog'}, ...}
```

Updated to be:
```
{'slug_name': ["dog"], ...}
```
This is to be more consistent with event based when it comes time to doing embeddings.